gateway/gateway.go

```diff
@@ -447,10 +447,18 @@ func (gateway *HandleT) userWebRequestWorkerProcess(userWebRequestWorker *userWe
 				}
 				continue
 			}
-			jobList = append(jobList, jobData.job)
-			jobIDReqMap[jobData.job.UUID] = req
-			jobSourceTagMap[jobData.job.UUID] = sourceTag
-			eventBatchesToRecord = append(eventBatchesToRecord, sourceDebugger{data: jobData.job.EventPayload, writeKey: writeKey})
+			jobList = append(jobList, jobData.jobs...)
+			lo.ForEach(jobData.jobs, func(job *jobsdb.JobT, _ int) {
+				jobIDReqMap[job.UUID] = req
+				jobSourceTagMap[job.UUID] = sourceTag
+				eventBatchesToRecord = append(
+					eventBatchesToRecord,
+					sourceDebugger{
+						data:     job.EventPayload,
+						writeKey: writeKey,
+					},
+				)
+			})
```

Comment:
[minor] What is the benefit of using `lo.ForEach` instead of `for`?

```suggestion
			for _, job := range jobData.jobs {
				jobIDReqMap[job.UUID] = req
				jobSourceTagMap[job.UUID] = sourceTag
				eventBatchesToRecord = append(
					eventBatchesToRecord,
					sourceDebugger{
						data:     job.EventPayload,
						writeKey: writeKey,
					},
				)
			}
```

Reply:
no benefit really, I've just become used to it.